connector/connect/common/src/main/protobuf/spark/connect/catalog.proto

```diff
@@ -77,6 +77,8 @@ message ListDatabases {
 message ListTables {
   // (Optional)
   optional string db_name = 1;
+  // (Optional) The pattern that the table name needs to match
+  optional string pattern = 2;
```

Comment:
Do we need a test case for Spark Connect?

Reply:
I added test cases in `CatalogSuite`.